hal/include/FRC_NetworkCommunication/UsageReporting.h

```diff
@@ -91,7 +91,7 @@ namespace nUsageReporting
         kCANPlugin_2CAN = 2,
 
         kFramework_Iterative = 1,
-        kFramework_Simple = 2,
+        kFramework_Sample = 2,

```

Comment:
This is a generated file.  We should fix the uses rather than change anything here until it's fixed upstream.